jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -40,7 +40,9 @@ import LoadingDots from '../../utils/components/LoadingDots/LoadingDots';
 import FullPageError from './components/FullPageError/FullPageError';
 import {requestAPI} from '../../handler';
 
-export const MOUNT_BROWSER_NAME = 'mount-browser:';
+export const MOUNT_BROWSER_PREFIX = 'mount-browser-';
+export const PFS_MOUNT_BROWSER_NAME = 'mount-browser-pfs:';
```

Comment:
Can we assign these two names from the created file browsers rather than setting them ahead of time? That way if the naming logic when creating the file browser changes, we don't need to be aware of having to change these too.

Reply:
do you mind elaborating what you mean by this? do you mean to get rid of the `mount-browser` prefix and just name the mount browsers themselves?

Reply:
Instead of hardcoding the names for these pfs and datum mount browser name variables, can we derive the values to assign these variables from the two browser objects we create further below?

Reply:
oh i see what you mean. it's not that straightforward since the jupyter `FileBrowser` that we implement doesn't expose its name. having a const and using that everywhere seems to be the next best thing to me, let me know if i'm missing something.